packages/manager/apps/web/client/app/private-database/index.js

```diff
@@ -0,0 +1,20 @@
+import controller from './private-database.controller';
+import routing from './private-database.routing';
+import service from './private-database.service';
+import tabController from './private-database-tabs.controller';
+
+import orderCloudDb from './order/clouddb';
+
+const moduleName = 'ovhManagerWebPrivateDatabaseModule';
+
+angular
+  .module(moduleName, [
+    orderCloudDb,
+  ])
+  .config(routing)
+  .controller('PrivateDatabaseCtrl', controller)
+  .controller('PrivateDatabaseTabsCtrl', tabController)
```

Comment:
Is this still needed ? Maybe we could replace by `oui-header-tabs`

Reply:
Still needed yes, we could, but this requires more refactor.